hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/DynamicConfigYamlGenerator.java

```diff
@@ -965,6 +966,15 @@ private static List<Map<String, Object>> getIndexConfigsAsList(List<IndexConfig>
                         indexConfig.getBitmapIndexOptions().getUniqueKeyTransformation().name());
 
                 indexConfigAsMap.put("bitmap-index-options", bitmapIndexOptionsAsMap);
+            } else if (indexConfig.getType() == IndexType.SORTED) {
+                BTreeIndexConfig bTreeConf = indexConfig.getBTreeIndexConfig();
+                Map<String, Object> btreeOptionsAsMap = new LinkedHashMap<>();
+                addNonNullToMap(btreeOptionsAsMap, "page-size", getCapacityAsMap(bTreeConf.getPageSize()));
+
+                Map<String, Object> capacityAsMap = new LinkedHashMap<>();
+                addNonNullToMap(capacityAsMap, "capacity", getCapacityAsMap(bTreeConf.getMemoryTierConfig().getCapacity()));
+
+                addNonNullToMap(btreeOptionsAsMap, "memory-tier", capacityAsMap);
```

Comment:
What do we this btreeOptionsAsMap? I see we have populated it, but didn't put it to anywhere

Reply:
Can we add a test for this to AbstractDynamicConfigGeneratorTest?

Reply:
Added, thx